src/main/java/io/wisetime/connector/config/RuntimeConfig.java

```diff
@@ -86,6 +86,16 @@ private RuntimeConfig() {
     return INSTANCE.getString(configKey.getConfigKey());
   }
 
+  public static void setProperty(RuntimeConfigKey configKey, String value) {
+    System.setProperty(configKey.getConfigKey(), value);
```

Comment:
There is unused map in configuration: https://github.com/wisetime-io/wisetime-connector-java/blob/b7363516c60b219d8f7bd773b63990113368640d/src/main/java/io/wisetime/connector/config/RuntimeConfig.java#L42 From my point the in-memory map suits much better for setProperty(), clearProperty(). With current implementation (usage `System.setProperty()`) there is always a risk to forget clean up some property and rebuild config do nothing. E.g.:
```java
//Jira connector implemention (or other test)
@Test
public void testExtendedOutput() {
  RuntimeConfig.setProperty("EXTENDED_NARRATIVE", "true");
  //test posted time has extended description
  //if   RuntimeConfig.clearProperty("EXTENDED_NARRATIVE"); was not called after test it may effect 
  //other tests later and different test order produce different results which is not easy to investigate
  //RuntimeConfig.rebuild(); - do nothing
  // user is forced to manually undo every config change he set in test - otherwise suffer from
  // mysterious failures in another tests
}
```
If instead of updating system properties in set/clearProperty() we will write to the in-memory map - its easy to reset all config changes: RuntimeConfig.rebuild(); will create the new instance with new empty config map.

Reply:
@sukolenvo agree, the approach we've used previously via the in-memory map is a better implementation.  Can you bring it over ?